frontend-react/src/components/MessageTracker/MessageReceivers.tsx

```diff
@@ -271,7 +271,10 @@ export const MessageReceiversRow = ({
 //     fileName: string;
 //     transportResults: string;
 // }
-export function formatMessages(data: ReceiverData[]): NormalizedReceiverData[] {
+export function formatMessages(
```

Comment:
Rather than update our source code to work around the bug in Node v18.13, can we just lock our version down to 18.12.1 for now?  It looks like [the Node v18.15 proposal](https://github.com/nodejs/node/pull/46920) includes [the fix](https://github.com/nodejs/node/commit/1d5058d4bb23897e933aad021e30854e111aff5b) so we can just upgrade directly to that after that's released.  Whatchu think?

Reply:
18.12.1 is likely vulnerable from the CVE's that forced us to update the 16.x branch last month (see: https://nodejs.org/en/blog/vulnerability/february-2023-security-releases/)

Reply:
We also still need to be ready for when changes like this happen again in the future for Intl strings as browser vendors expect devs to assume Intl strings can change at any time, unless we're deeming this as a puntable issue.

Reply:
The security vulnerabilities in 18.12.1 are tolerable in my opinion because we're not running Node anywhere in production; all the JavaScript that we run in production is in the browser.  But actually, this might be a moot point since 18.15 looks like it just released today?

Regarding the future of formatter strings, my opinion is that it isn't good practice to structure source code around test -- like if there weren't a difference between browser and Node implementation, we wouldn't have to pass in a `formatter` argument to `formatMessages`.  And if we were using this formatter in a lot of different places in the code, the amount of manual maintenance would add up.  

Since there was enough developer outrage to cause both browsers and Node to revert -- even if temporarily -- my vote here would be to wait to see what happens in the wider community.  Even if browsers undo their reversion before Node, it wouldn't be a breaking (pun intended?) change; the worst that would happen would be that we wouldn't have line breaks in formatted strings.  Then we'd just have to update our tests whenever Node undoes their reversion.

What are your thoughts?

Reply:
> it isn't good practice to structure source code around test

+1 to this!